test/commands/dev/index.test.ts

```diff
@@ -1170,6 +1170,32 @@ describe('local dev environment', function () {
         expect(runCompose.firstCall.args[0]).to.deep.equal(expected_instance_compose);
       });
 
+    local_dev
+      .stub(Dev.prototype, 'runCompose', sinon.stub().returns(undefined))
+      .stub(Dev.prototype, 'downloadSSLCerts', sinon.stub().returns(undefined))
+      // @ts-ignore
```

Comment:
What are we ignoring here and why?

Reply:
I copied from the other tests. It seems to yield an argument type error without it.
```
error TS2345: Argument of type '(_: any, metadata: any) => ComponentSpec' is not assignable to parameter of type '() => any'
```